geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1561,11 +1564,12 @@ public Object selectValue(String queryPredicate) throws FunctionDomainException,
    * events to your client cache.
    * </p>
    *
-   * @param key The key on which to register interest. If the key is a <code>List</code>, then all
-   *        the keys in the <code>List</code> will be registered. The key can also be the special
-   *        token 'ALL_KEYS', which will register interest in all keys in the region. In effect,
-   *        this will cause an update to any key in this region in the CacheServer to be pushed to
-   *        the client.
+   * @param key The key on which to register interest.
+   *
+   *        deprecated behavior If the key is a <code>List</code>, then all the keys in the
```

Comment:
I like how you marked the beginning and end of the deprecated behavior in registerInterest like:

The following <code>List</code> and 'ALL_KEYS' behavior is now deprecated:
...
End of deprecation

Do you want to do that here too?